{
  "cve_id": "CVE-2023-50693",
  "github_data": {
    "repository": "dom96/jester",
    "fix_commit": null,
    "related_commits": [],
    "patch_url": null
  },
  "vulnerability_details": {
    "cvss_score": 9.8,
    "cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
    "cwe_id": null,
    "description": "An issue in Jester v.0.6.0 and before allows a remote attacker to send a malicious crafted request.",
    "attack_vector": "NETWORK",
    "attack_complexity": "LOW"
  },
  "temporal_data": {
    "published_date": "2024-01-19T20:15:11.917",
    "last_modified": "2024-11-21T08:37:09.380",
    "fix_date": null
  },
  "references": [
    {
      "url": "https://gist.github.com/anas-cherni/dd297786750f300a2bab3bb73fee919b",
      "source": "cve@mitre.org",
      "tags": [
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://github.com/dom96/jester/issues/326",
      "source": "cve@mitre.org",
      "tags": [
        "Exploit",
        "Issue Tracking"
      ]
    },
    {
      "url": "https://github.com/dom96/jester/pull/327",
      "source": "cve@mitre.org",
      "tags": [
        "Issue Tracking"
      ]
    },
    {
      "url": "https://gist.github.com/anas-cherni/dd297786750f300a2bab3bb73fee919b",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://github.com/dom96/jester/issues/326",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Exploit",
        "Issue Tracking"
      ]
    },
    {
      "url": "https://github.com/dom96/jester/pull/327",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Issue Tracking"
      ]
    }
  ],
  "collection_metadata": {
    "collected_at": "2025-01-11T23:07:05.509804",
    "processing_status": "raw"
  }
}